system/system.proto

```diff
@@ -96,8 +96,10 @@ message SwitchControlProcessorRequest {
 
 message SwitchControlProcessorResponse {
   types.Path control_processor = 1;
```

Comment:
you might also call it out that the control_processor is about the target processor too?

Reply:
@xw-g done in [7846102](https://github.com/openconfig/gnoi/pull/114/commits/784610276dad74a47c5d38d740882fb0efb73745)